app/angular/src/server/utils/filter-out-styling-rules.ts

```diff
@@ -0,0 +1,20 @@
+import { Configuration, RuleSetRule } from 'webpack';
+
+const isStylingRule = (rule: RuleSetRule) => {
+  const { test } = rule;
+
+  if (!test) {
+    return false;
+  }
+
+  if (!(test instanceof RegExp)) {
+    return false;
+  }
+
+  return test.test('.css') || test.test('.scss') || test.test('.sass');
+};
+
+export const filterOutStylingRules = (config: Configuration) => {
+  // @ts-ignore
```

Comment:
What needs to be ignored here? 🤔

Reply:
Hummm 🤔  I took it from the old code. I'll go check it out if it's still nessesary